.github/workflows/e2e-tests.yml

```diff
@@ -92,6 +92,13 @@ jobs:
         run_id: ${{ github.event.workflow_run.id }}
         path: build-output
 
+    - name: Checkout mirror repo
+      if: github.event_name == 'repository_dispatch'
+      uses: actions/checkout@v3
+      with:
+        repository: ${{ github.event.client_payload.repository }}
+        path: build-output/build/${{ github.event.client_payload.repository }}
```

Comment:
Use the same path created by downloading the build workflow artefacts, so we can use the same `map-plugins-for-e2e-env.sh` script